os/net/ipv6/uipopt.h

```diff
@@ -547,6 +531,19 @@ void uip_log(char *msg);
  */
 #define UIP_TC_MAC_TRANSMISSION_COUNTER_MASK 0x3F
 
+#ifdef UIP_CONF_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS
+#define UIP_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS UIP_CONF_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS
+#else
+#define UIP_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS 0
+#endif
+
+/**
+ * This is the default value of MAC-layer transmissons for uIPv6
+ *
+ * It means that the limit is selected by the MAC protocol instead of uIPv6.
+ */
+#define UIP_MAX_MAC_TRANSMISSIONS_UNDEFINED 0
```

Comment:
Let me rephrase my concern here. If we define this macro to a different value, things will break, because there is no `uipbuf_set_attr(UIPBUF_ATTR_MAX_MAC_TRANSMISSIONS, UIP_MAX_MAC_TRANSMISSIONS_UNDEFINED)`.

Reply:
Ok, we can move this into the uip clearbuf so that it is set to this value there?

Reply:
works for me!